### PR TITLE
Issue #1662 #1706 Profile related messages during start and delete

### LINKED ISSUE #1662: No indication of profile being started
I would like to have as first line of the logs when you start, something like:

```
-- Starting minishift profile XYZ
```

### PR DESCRIPTION
**Pull request by ulyssessouza**
Fixes #1662, #1706 

 * Add profile name into the start message
 * Consolidate delete messages

### CODE REVIEW

File: cmd/minishift/cmd/start.go [hunk 1]
```diff
@@ -347,7 +347,7 @@ func startHost(libMachineClient *libmachine.Client) *host.Host {
 
 	cacheMinishiftISO(machineConfig)
 
-	fmt.Print("-- Starting Minishift VM ...")
+	fmt.Printf("-- Starting Minishift VM profile '%s'...", minishiftConfig.AllInstancesConfig.ActiveProfile)
```
Comment (gbraad) [inline cmd/minishift/cmd/start.go]: This is in my opinion too late... several configuration messages have already been shown at this time, such as:

```
-- Starting local OpenShift cluster using 'hyperv' hypervisor ...
-- Minishift VM will be configured with ...
   Memory:    4 GB
   vCPUs :    2
   Disk size: 10 GB
```
And even on Hyper-V.
```
-- Attempting to set network settings ...
   Device:      eth0
   IP Address:  10.0.75.128/24
   Gateway:     10.0.75.1
   Nameservers: 8.8.8.8
```

In my opinion this should be one of the first messages.

**Comment by gbraad** [comment]
Thanks for the contribution, but some minor changes are needed.

**Comment by gbraad** [comment]
I changed the description and title of the PR to reflect the additional commit

**Comment by gbraad** [comment]
First, this should be:
```diff
- -- Starting Minishift VM profile 'minishift'......................... OK
+ -- Starting Minishift VM profile 'minishift' ......................... OK
```
as this would be consistent with all the other messages.

But I even believe it should be earlier in the process. As you can see in the request, it should be one of the first messages in the log:

```diff
+ -- Starting profile 'minishift'
  -- Starting local OpenShift cluster using 'kvm' hypervisor ...
  -- Minishift VM will be configured with ...
   Memory:    2 GB
   vCPUs :    2
   Disk size: 20 GB
- -- Starting Minishift VM profile 'minishift'......................... OK
+ -- Starting Minishift VM ......................... OK
```

We can always consolidate messages again at a later stage. For now, verbosity is not a very big problem... as long as it is meaningful information.

**Comment by gbraad** [comment]
new PR created as #1710 ?

**Comment by LalatenduMohanty** [comment]
+1 for creating two separate PRs for each issue.

**Pull request closed without merging**